lighthouse-core/report/v2/report-styles.css

```diff
@@ -739,6 +745,7 @@ summary.lh-passed-audits-summary::-webkit-details-marker,
 @media screen and (max-width: 767px) {
   .lh-report {
     margin-left: 0;
+    width: 100%;
```

Comment:
i think we should one min-width on `lh-report` or similar container...

to avoid this sorta thing

![narrow](https://cloud.githubusercontent.com/assets/39191/26183487/d77e9774-3b33-11e7-8ed3-de1a2df3ba52.gif)